cnd/src/hbit.rs

```diff
@@ -6,14 +6,14 @@ use crate::{
     LocalSwapId, Role, Side,
 };
 use bitcoin::{Address, Block, BlockHash};
-use chrono::{DateTime, Utc};
 use comit::{asset, htlc_location, transaction, LockProtocol, Secret};
 pub use comit::{hbit::*, identity};
 use futures::TryStreamExt;
 use std::{
     collections::{hash_map::Entry, HashMap},
     sync::Arc,
 };
+use time::OffsetDateTime;
```

Comment:
Do we have the same guarantees with `OffsetDateTime` than we use to have? e.g. ensuring there is no mix with local and utc?

Also, why not using `chrono` instead of `time`? You did not justify the choice.

Reply:
> Do we have the same guarantees with `OffsetDateTime` than we use to have? e.g. ensuring there is no mix with local and utc?

I am 100% sure but I think so. `OffsetDateTime` can be constructed with a specific offset but there is no guarantee that it was constructed with a specific one. It shouldn't matter much though because the instance does track the timezone and within `btsieve` we only compare unix timestamps which are always in UTC.

The problem we had previously was that we constructed a `DateTime` WITHOUT timezone information.

> Also, why not using `chrono` instead of `time`? You did not justify the choice.

Usage of `time` is more spread out in the codebase due to @tcharding `expiries` module, hence moving to `time` was easier than the other way round. Also, development of `time` seems to be more active and it is the more lightweight dependency.

I am not too fussed about it though, we can move back to `chrono` if we want at some point. I just wanted it to be uniform :)